helix-core/src/main/java/org/apache/helix/common/caches/BasicClusterDataCache.java

```diff
@@ -38,8 +37,32 @@
 public class BasicClusterDataCache {
   private static Logger LOG = LoggerFactory.getLogger(BasicClusterDataCache.class.getName());
 
-  protected Map<String, LiveInstance> _liveInstanceMap;
-  protected Map<String, InstanceConfig> _instanceConfigMap;
+  private static final String INSTANCE_CONFIG = "InstanceConfig";
+  private static final String LIVE_INSTANCE = "LiveInstance";
+  private static final ControlContextProvider DUMMY_CONTROL_CONTEXT_PROVIDER =
```

Comment:
maybe a better name?  DUMMY seems not so serious in prod code?

Reply:
Sure.

Reply:
maybe just call it BASIC_CONTEXT_PROVIDER?

Reply:
you can call AbstractDataCache.createDefaultControlContextProvider() to create the default provider.